pkg/sriov/sriov_test.go

```diff
@@ -388,23 +401,20 @@ var _ = Describe("Sriov", func() {
 			mocked.On("LinkSetNsFd", fakeLink, mock.AnythingOfType("int")).Return(nil)
 			mocked.On("LinkSetUp", fakeLink).Return(nil)
 			sm := sriovManager{nLink: mocked}
```

Comment:
you should mock out the utils as well. I think that is we are seeing failure because it is called in your setupVF.